src/Selenium2Driver.php

```diff
@@ -107,45 +115,12 @@ public function setDesiredCapabilities($desiredCapabilities = null)
         }
 
         if (null === $desiredCapabilities) {
-            $desiredCapabilities = array();
+            $desiredCapabilities = [];
         }
 
         // Join $desiredCapabilities with defaultCapabilities
         $desiredCapabilities = array_replace(self::getDefaultCapabilities(), $desiredCapabilities);
 
-        if (isset($desiredCapabilities['firefox'])) {
```

Comment:
Seems that the facebook webdriver already does this process... I have to be sure though.